src/graphql/data/nft/CollectionSearch.ts

```diff
@@ -0,0 +1,88 @@
+import { isAddress } from '@ethersproject/address'
+import { useNftGraphqlEnabled } from 'featureFlags/flags/nftlGraphql'
+import gql from 'graphql-tag'
+import { GenieCollection } from 'nft/types'
+import { blocklistedCollections } from 'nft/utils'
+import { useMemo } from 'react'
+
+import { NftCollection, useCollectionSearchQuery } from '../__generated__/types-and-hooks'
+import { formatCollectionQueryData, useCollection } from './Collection'
+
+const MAX_SEARCH_RESULTS = 6
```

Comment:
are the max search results not 5? `slice(0, 5)` will return 5 items - or did we change the spec to 6?

Reply:
Max is 6, you can be shown up to 6 if there are less than 3 token results